<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelLoadingModeDialog</class>
 <widget class="QDialog" name="ModelLoadingModeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model Loading Mode Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>286</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="defaultRadio">
      <property name="text">
       <string>Default</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="useOSGBRadio">
      <property name="text">
       <string>Use existing OSGB format</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="buildOSGBRadio">
      <property name="text">
       <string>Build an use OSGB format</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="useSmartLODRadio">
      <property name="text">
       <string>Use existing SmartLOD files</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="buildSmartLODRadio">
      <property name="text">
       <string>Build and use SmartLOD files</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="useTilesRadio">
      <property name="text">
       <string>Use existing Tiles</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="useTilesFolderRadio">
      <property name="text">
       <string>Use existing Tiles in folder</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="useSmartLODTilesRadio">
      <property name="text">
       <string>Use existing SmartLOD Tiles</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="useSmartLODTilesFolderRadio">
      <property name="text">
       <string>Use existing SmartLOD Tiles in folder</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="buildSmartLODTilesRadio">
      <property name="text">
       <string>Build and use SmartLOD Tiles</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>238</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="saveCompoudCheck">
      <property name="text">
       <string>Save also compound LOD files</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="Xlabel">
        <property name="text">
         <string># Tiles in X :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="XspinBox">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="Ylabel">
        <property name="text">
         <string># Tiles in Y :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="YspinBox">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>281</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
